akka-http/src/main/scala/akka/http/javadsl/server/Directives.scala

```diff
@@ -26,4 +28,96 @@ object Directives extends AllDirectives {
 
   @varargs override def getFromBrowseableDirectories(directories: String*): Route =
     super.getFromBrowseableDirectories(directories: _*)
+
+  /**
+   * Partially applying functions for the Java World!
```

Comment:
I would say it's partially application: https://en.wikipedia.org/wiki/Partial_application
>Intuitively, partial function application says "if you fix the first arguments of the function, you get a function of the remaining arguments"